shopware/paas-meta/6.4/.platform/applications.yaml

```diff
@@ -205,9 +211,17 @@
             disk: 128
             commands:
                 start: |
-                    bin/console messenger:consume default --memory-limit=$(cat /run/config.json | jq .info.limits.memory)M --time-limit=295
+                    bin/console cache:clear
```

Comment:
why do we need to run `cache:clear` before?

Reply:
I've noticed some issues after a deploy that old cache is still used, based on old project code. By clearing it before running, this should be fixed. As far as I could see, there is no way to run the cache clear on workers just once after deploys.

So I agree that it's not ideal, but I think it is necessary.